.github/dependabot.yml

```
@@ -8,4 +8,14 @@ updates:
     open-pull-requests-limit: 10
     target-branch: main
     labels:
-      - dependabot
\ No newline at end of file
+      - dependabot
+  - package-ecosystem: docker
+    directory: /
```

Nitpick but for consistency with above (or remove from above)
```suggestion
    directory: "/"
```